controllers/registration.js

```diff
@@ -71,23 +71,22 @@ exports.registrationUserPost = async (req, res) => {
                 { _id: workPlace },
                 {
                     $push: {
-                        medStaff: doctor.id,
+                        medStaff: doctor._id,
                     },
                 }
             );
         }
-        const token = jwt.sign({ id: user.id }, config.get("secretKey"), {
-            expiresIn: "1h",
-        });
-        return res.status(201).send({
-            token,
-            user: {
-                id: user.id,
-                name: user.name,
-                userRole: user.userRole,
-                photo: user.photo,
-            },
-        });
+        updateTokens(user.id).then((tokens) =>
+            res.status(200).send({
+                tokens,
```

Comment:
why sending tokens in response body?

Reply:
fixed, access and refresh token send to the client in cookies now